app/controllers/user_controller.rb

```diff
@@ -16,6 +20,16 @@ def show
     end
   end
 
+  def show_theme
+    theme = Theme.where(user: User.where("LOWER(screen_name) = ?", params[:username].downcase).select(:id)).first
+    return head :no_content if theme.nil?
```

Comment:
using `unless` here is a bit more idiomatic
```suggestion
    return head :no_content unless theme
```